jOOR-java-8/src/main/java/org/joor/Reflect.java

```diff
@@ -103,6 +103,9 @@ public static Reflect compile(String name, String content, CompileOptions option
         return onClass(Compile.compile(name, content, options));
     }
 
+    public static Reflect compileSkipLoaded(String name, String content, CompileOptions options) throws ReflectException {
+        return onClass(Compile.compile(name, content, options, true));
+    }
```

Comment:
I personally don't think this flag is significant enough to warrant new API. It's a boolean flag and if we ever wanted to add new boolean flags, we'd have to duplicate each one of these methods to allow for all the possible combinations of these flags.

That's why we've added `CompileOptions`. The flag should probably go in there.